src/models/inline.js

```diff
@@ -14,6 +13,7 @@ import Block from './block'
 import Data from './data'
 import Node from './node'
 import Text from './text'
+import TYPES from './types'
```

Comment:
Given the file name change (from another comment), can we change all of these `TYPES` variable names to `MODEL_TYPES`? Thanks!